wolfssl/wolfcrypt/sp_int.h

```diff
@@ -95,7 +95,9 @@ extern "C" {
     #error "Size of unsigned int not detected"
 #endif
 
-#if ULONG_MAX == 18446744073709551615UL
+#if ULONG_MAX == 18446744073709551615ULL && \
+        4294967295UL != 18446744073709551615ULL /* verify pre-processor supports
```

Comment:
Is this needed?

Reply:
Yes! Very much. On their C89 compiler the pre-processor does NOT support 64-bit ULL pre-processor, so this check was reporting TRUE, when it is false.

Reply:
Thanks!